2020/1-year.adoc

```diff
@@ -0,0 +1,44 @@
+= [Current Year] One Year Plan for the People's Liberation Training Committee
+
+:toc:
+:toc-placement!:
+
+toc::[]
+
+== Purpose
+
+* Summary of Goals
+
+* Summary of Actions
```

Comment:
```suggestion
// Summary of Goals

// Summary of Actions
```

If we're turning this into a template of sorts, using comments instead of bullets means the instructions will always be visible to the person doing the writing, and they wont have to delete them. 

depends on what you want it to do though. If you want this to be step-by-step instructions rather than a template, dont change this

Reply:
Yeah I was aiming for a step-by-step, but I'm still fine with commenting them for easier use